dashboard/package.json

```diff
@@ -29,13 +29,13 @@
     "react-lazylog": "^2.1.0",
     "react-markdown": "^3.1.4",
     "react-modal": "^3.1.11",
-    "react-redux": "^5.0.6",
+    "react-redux": "5.0.6",
     "react-router-dom": "^4.2.2",
     "react-router-redux": "^5.0.0-alpha.9",
     "react-scripts-ts": "^2.17.0",
     "react-select": "^1.2.1",
     "react-test-renderer": "^16.2.0",
-    "redux": "^3.7.2",
+    "redux": "4.0.0",
```

Comment:
don't you trust semver ranges?

Reply:
You were right. I had it locked because I was fighting with versions and stuff. Changed, thanks!